test/lsptoolshost/integrationTests/formatting.integration.test.ts

```diff
@@ -12,19 +12,19 @@ import {
     expectText,
     openFileInWorkspaceAsync,
 } from './integrationHelpers';
-import { describe, beforeAll, beforeEach, afterAll, test, afterEach } from '@jest/globals';
+import { describe, beforeEach, afterAll, test, afterEach } from '@jest/globals';
 import { formatDocumentAsync, formatOnTypeAsync, formatRangeAsync } from './formattingTestHelpers';
 
 describe(`Formatting Tests`, () => {
-    beforeAll(async () => {
+    beforeEach(async () => {
+        await setOrganizeImportsOnFormat(undefined);
         await activateCSharpExtension();
```

Comment:
Dropped the `beforeAll` and moved the activation so that it occurred after we reset the organize option.